lib/ash_json_api/resource/resource.ex

```diff
@@ -350,7 +350,7 @@ defmodule AshJsonApi.Resource do
     examples: [
       """
       routes do
-        base_route "/posts"
+        base "/posts"
```

Comment:
Got compilation errors following these examples. The schema above defines `base`, which got it working.